src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxOptionsType.php

```diff
@@ -74,16 +82,58 @@ public function __construct(
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('enable_tax', SwitchType::class)
-            ->add('display_tax_in_cart', SwitchType::class)
+        $builder->add(
+            'enable_tax',
+            SwitchType::class,
+            [
+                'label' => $this->trans('Enable tax', 'Admin.International.Feature'),
+                'help' => $this->trans(
+                    'Select whether or not to include tax on purchases.',
+                    'Admin.International.Help'
+                ),
+                'required' => false,
+                'attr' => [
+                    'class' => 'js-enable-tax',
+                ],
+            ]
+        )
+            ->add('display_tax_in_cart', SwitchType::class, [
```

Comment:
You should write as you did for 'enable_tax', the indentation is weird.

Reply:
Sorry, I didn't notice the description hasn't been changed, but the label was added :wink: